doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+
+    echo "Check ACRN efi boot event"
+    check_acrn_bootefi=`efibootmgr | grep ACRN`
+    if [[ "$check_arcn_bootefi" -ge "ACRN" ]]; then
+        echo "Clean all ACRN efi boot event"
+        efibootmgr | grep ACRN | cut -d'*' -f1 | cut -d't' -f2 | xargs -i efibootmgr -b {} -B
+    fi
+    echo "Check linux bootloader event"
+    number=$(expr `efibootmgr | grep 'Linux bootloader' | wc -l` - 1)
+    if [[ $number -ge 1 ]]; then
+        echo "Clean all Linux bootloader event"
+        efibootmgr | grep 'Linux bootloader' | cut -d'*' -f1 | cut -d't' -f2 | head -n$number | xargs -i efibootmgr -b {} -B
+    fi
+
+    echo "Add new ACRN efi boot event"
+    efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN"
+
+    echo "Create loader.conf"
+    rm /mnt/loader/loader.conf && touch /mnt/loader/loader.conf
+    echo "Add default boot wait time"
+    echo "timeout 5" > /mnt/loader/loader.conf
+    echo "Add default boot to ACRN"
+    echo "default acrn" >> /mnt/loader/loader.conf
+
+    echo "Get latest sos file from EFI org.clearlinux: "
+    new_kernel=`ls /mnt/EFI/org.clearlinux/*sos* -tl | grep kernel | head -n1 | awk -F'/' '{print $5}'`
+    echo "Get current sos kernel from acrn.conf file: "
+    cur_kernel=`cat /mnt/loader/entries/acrn.conf | sed -n 2p | cut -d'/' -f4`
+
+    echo "Replace root uuid from acrn.conf"
+    sed -i "s/<UUID of rootfs partition>/`blkid -s PARTUUID -o value /dev/sda3`/g" /mnt/loader/entries/acrn.conf
+
+    if [[ "$cur_kernel" == "$new_kernel" ]]; then
+        echo "No need to replace kernel conf info"
+    else
+        echo "Replace with new sos kernel in acrn.conf"
+        sed -i "s/$cur_kernel/$new_kernel/g" /mnt/loader/entries/acrn.conf
+    fi
+
+    echo "Sos process done!"
+    echo " ***** Please reboot device to test on new sos. *****"
+else
+    echo -e "Fail to upgrade sos $target_clearlinux from mirror url."
+fi
+
+echo "Init swupd mirror & proxy"
+swupd mirror -u
+unset https_proxy
+umount /mnt
+}
+
+function upgrade_uos()
+{
+# Add proxy if you need to download uos img automatically.
+if [[ -n $proxy ]]; then
+    echo "Add proxy..."
+    export https_proxy=$proxy
+fi
+
+# Do upgrade uos process.
+if [[ $skip_download_uos == 1 ]]; then
+    uos_img_xz_path=$(find ~/ -name clear-$target_version-kvm.img.xz)
+    uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+    if [[ ! -f $uos_img_xz_path ]] && [[ ! -f $uos_img_path ]]; then
+        echo "You should download uos clear-$target_version-kvm.img.xz file firstly." && exit
+    fi
+    if [[ -f $uos_img_xz_path ]]; then
+        echo "get uos xz file in: $uos_img_xz_path"
+        unxz $uos_img_xz_path
+        uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+    fi
+    echo "get uos img file in: $uos_img_path"
+    if [[ $? -eq 0 ]]; then
+        losetup -f -P --show $uos_img_path
+    else
+        echo "fail to losetup uos img, please check uos img status" && exit 1
+    fi
+    mount /dev/loop0p3 /mnt
```

Comment:
Remove the hardcoded loop0 here. You need to read it from the previous command, that's what the --show flag from losetup will do for you, print out the loop device that has been used.

Reply:
You should also verify that nothing is already mounted under `/mnt`

Reply:
Yes, it should be handled in latest code.